nucypher/crypto/powers.py

```diff
@@ -490,6 +497,38 @@ def encrypt_decryption_response(
             raise self.ThresholdResponseEncryptionFailed from e
 
 
+class SigningRequestDecryptingPower(ThresholdRequestPower):
+    def decrypt_encrypted_request(
+        self,
+        encrypted_request: EncryptedThresholdDecryptionRequest,  # EncryptedThresholdSigningRequest
+    ) -> ThresholdDecryptionRequest:  # ThresholdSigningRequest:
```

Comment:
Ah, I see the short coming now of waiting for my `nucypher-core` changes... I missed this need. Perhaps don't put any typehints for now? (Same for the encrypt_decryption_response method below)

wdyt?